hack/pp-cm-helper.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+ENV_FILE=/tmp/cm.env
```

Comment:
```suggestion
ENV_FILE=$(mktemp -t pp-cm-env.XXXXX)
```
Just to be 100% that two concurrent instances of this script don't step on each other's foot.